geometry_msgs/Point pose # z coordinate is the orientation
geometry_msgs/Vector3 twist
string id